SmartDeviceLinkTests/DevAPISpecs/SDLStreamingVideoLifecycleManagerSpec.m

```diff
@@ -713,6 +799,129 @@ __block void (^sendNotificationForHMILevel)(SDLHMILevel hmiLevel, SDLVideoStream
         });
     });
 
+    describe(@"when the manager is stopped", ^{
+        beforeEach(^{
+            streamingLifecycleManager.connectedVehicleMake = @"OEM_make";
```

Comment:
why? It is just a test case.

Reply:
Because if there's another test case that it needs to be used, or if it needs to be checked in an expect, it's easier to have it be in a var or constant.